extensions/ql-vscode/src/databases/config/db-config-store.ts

```diff
@@ -459,6 +460,7 @@ export class DbConfigStore extends DisposableObject {
 
   private createEmptyConfig(): DbConfig {
     return {
+      version: SUPPORTED_DB_CONFIG_VERSION,
```

Comment:
See comment below, I would expect an array of all supported versions if we use this constant name.